build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.5
+FROM golang:1.5.1
```

Comment:
Hmm, good point. https://circleci.com/gh/cockroachdb/cockroach/8141 failed last night and that only occurs on 1.5. I'll investigate the necessity.